pkg/kubelet/util/pod_startup_latency_tracker_test.go

```diff
@@ -141,6 +141,15 @@ kubelet_pod_start_sli_duration_seconds_count 1
 		fakeClock.Step(time.Millisecond * 100)
 		tracker.RecordImageFinishedPulling(podInit.UID)
 
+		podState, ok := tracker.pods[podInit.UID]
+		if !ok {
+			t.Errorf("expected to track pod: %s, but pod not found", podInit.UID)
+		}
+		if !podState.lastFinishedPulling.Equal(podState.firstStartedPulling.Add(time.Millisecond * 100)) {
+			t.Errorf("expected pod firstStartedPulling: %s and lastFinishedPulling: %s but got firstStartedPulling: %s and lastFinishedPulling: %s",
```

Comment:
nit: `frozenTime` is effectively the same as `podState.firstStartedPulling`, but can we keep the error message consistent with the equal check? i.e.: 

```
t.Errorf("expected pod firstStartedPulling: %s and lastFinishedPulling: %s but got firstStartedPulling: %s and lastFinishedPulling: %s",
podState.firstStartedPulling, podState.firstStartedPulling.Add(time.Millisecond*100), podState.firstStartedPulling, podState.lastFinishedPulling)
```